### PR TITLE
Rename sheet when puzzle is solved

### LINKED ISSUE #135: Rename puzzle spreadsheet as [SOLVED: THEANSWER] ... when puzzle is solved
Might be a useful way to notify people working on the puzzle that it's been solved, in case they missed the Slack message and didn't notice on the site and nobody told them.

### PR DESCRIPTION
**Pull request by rgossiaux**
Fixes #135

Didn't put the answer in there since multiple answers makes it a little
more complicated; figure this is the important info anyway.

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -71,6 +71,12 @@ def create(self, request, **kwargs):
                 transaction.on_commit(
                     lambda: AnswerViewSet.__update_meta_sheets_for_feeder(puzzle)
```
Comment (asdfryan) [inline api/views.py:72]: Come to think of it -- this line should also be wrapped by `if google_api_lib.enabled()`.

Comment (rgossiaux) [reply api/views.py:72]: Aha hmm-- `update_meta_sheets_for_feeder` used to have a check for this inside and it looks like as part of #295 @rawxfish moved the check inside the GoogleApiClientTask __init__ in google_api_lib/utils.py.

Comment (rgossiaux) [reply api/views.py:72]: Testing locally I can confirm that the celery worker complains so I'll just fix on this branch.

Comment (rgossiaux) [reply api/views.py:72]: Split it off for clarity: #338

Comment (rawxfish) [reply api/views.py:72]: Whoops sorry about that

**Review by rawxfish — APPROVED** [review]

**Pull request merged**